examples/vscode-sandbox/README.md

```
@@ -196,10 +196,21 @@ If you are using gVisor or Kata Containers, direct pod port-forwarding isn't com
 
 2.  **Deploy the Gateway (Production Only):**
-    If you need external access via a Public IP (GKE), apply the Gateway configuration.
+    If you need external access via a public IP, apply the Gateway configuration.
+    This requires the Gateway API CRDs and a Gateway API controller
+    (e.g. Istio, GKE built-in, Envoy Gateway) — see the
+    [sandbox-router README](../../clients/python/agentic-sandbox-client/sandbox-router/README.md) for prerequisites.
+    Edit `spec.gatewayClassName` in the YAML to match your environment — see the
+    comments in `gateway.yaml` for options.
     ```bash
-    # Deploys Gateway, HTTPRoute, and HealthCheckPolicy
+<<<<<<< HEAD
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)